circuit_maintenance_parser/parsers/tata.py

```diff
@@ -0,0 +1,69 @@
+# pylint: disable=disallowed-name
+"""Circuit maintenance parser for Tata Email notifications."""
+from typing import List, Dict, Any
+from datetime import datetime
+
+from bs4.element import ResultSet  # type: ignore
+from circuit_maintenance_parser.output import Impact, Status
+from circuit_maintenance_parser.parser import Html, EmailSubjectParser
+
+
+class HtmlParserTata(Html):
+    """Custom Parser for HTML portion of Tata circuit maintenance notifications."""
+
+    def parse_html(self, soup: ResultSet) -> List[Dict]:
+        """Parse Tata circuit maintenance email."""
+        prev: str = ""
+        data: Dict[str, Any] = {
+            "account": "N/A",
+            "circuits": [],
+            "organizer": soup.select("a[href^=mailto]")[0].text.strip(),
+        }
+        for span in soup.find_all("span"):
+            curr = span.text.strip()
+            if curr != prev:
+                if prev.lower() == "ticket reference - tcl":
```

Comment:
you could create a variable and reuse it in the if statements
```suggestion
                prev_lower = prev.lower()
                if prev_lower == "ticket reference - tcl":
```